content/drafts/kubernetes_on_raspberry_pi.rst

```diff
@@ -0,0 +1,104 @@
+##########################
+Kubernetes on Raspberry Pi
+##########################
+
+:date: 2019-12-22
+:tags: project
+:category: Linux
+:slug: kubernetes-on-raspberry-pi
+:author: John Nduli
+
+
+`Kubernetes <https://kubernetes.io/>`_ is an open-source system for
+automating deployment, scaling, and management of containerized
+applications. `K3S <https://k3s.io/>` is a light-weight version of
+kubernetes that can run on weak hardware e.g raspberry pi, and provides
+most of the functionality of kubernetes.
+
+I had three raspberry pis from previous projects (one v2 and two v3s)
+lying around, so I decided to set up kubernetes and try it out. I used
+raspbian buster lite as the linux version on all the raspberry pis. The
```

Comment:
It's a good idea to add a link to the raspbian buster website for further reading.